TurboMinerVirtualTable.Unity/Assets/Scripts/Ui/Lobby/PlayerLabelsManager.cs

```diff
@@ -52,10 +52,10 @@ private void SetupUi()
         var position = new Vector3(PlayersList.transform.position.x, PlayersList.transform.position.y - PlayersLabels.Count * 45);
         var playerLabel = Instantiate(Resources.Load<PlayerLabel>("Prefabs/PlayerLabel"), position, Quaternion.identity, PlayersList.transform);
 
-        if(playerLabel.NameLabel.text == "")
-        {
-            playerLabel.NameLabel.text = "Player" + (PlayersLabels.Count + 1);
-        }
+        //if(playerLabel.NameLabel.text == "")
```

Comment:
jak zostawiasz zakomentowany kod to daj do niego komentarz czemu zostal zakomentowany:
// commented because...
a w tym przypadku nie wiem czemu to zostawiasz, ze niby w przyszlosci bedziemy chcieli tego uzyc?
Tak samo zostawianie metody ktora nie ma wywolania (i wiemy ze nie bedzie miala) tez jest bez sensu.
Wywal ten zakomentowany kod i ta metode ArePlayersNamesDifferent, to juz nigdy nie zostanie uzyte, a wlasnie w ten sposob potem w kodzie zostaja jakies enigmatyczne nieuzywane metody i zakomentowany kod